packages/payload/src/index.ts

```diff
@@ -1133,12 +1127,12 @@ export const getPayload = async (
     return cached.payload
```

Comment:
Awaiting the promise is already wrapped in try{} catch {}. Feels weird not to wrap this floating promise execution in the same try {} catch {}